doc/release-notes/10503-cvoc-hidden-html-fields.md

```diff
@@ -0,0 +1,8 @@
+## Release Highlights
+
+### Updates on Support for External Vocabulary Services
+
+#### Hidden HTML Fields
+
+To enlarge javascript possibilities of [:CVocConf](https://guides.dataverse.org/en/6.3/installation/config.html#cvocconf) configured service, the child fields of the configured field had been added to the dataset metadata view.
```

Comment:
Suggested change: External Controlled Vocabulary scripts, configured via [:CVocConf](https://guides.dataverse.org/en/6.3/installation/config.html#cvocconf), can now access the values of managed fields as well as the term-uri-field for use in constructing the metadata view for a dataset.

Reply:
Suggested change accepted and commit